prijateli_tree/app/static/css/custom.css

```diff
@@ -26,3 +26,21 @@ tr.empty-table {
     margin-top: 30px;
     text-align: center;
 }
+
+.red_text {
+    color: red;
+    font-size: 24px;
+}
+
+.blue_text {
+    color: #007bff;
+    font-size: 24px;
+}
+
+.red_color {
+    color: red;
+}
+
+.blue_color {
+    color: #007bff;
+}
```

Comment:
Use these classes for text instead.

Reply:
I'm not following, I used all of them for text

Reply:
Ahhh, sorry! I thought I pasted a link: https://getbootstrap.com/docs/5.3/customize/color/#colors